.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "modules/accord"]
 	path = modules/accord
-	url = https://github.com/apache/cassandra-accord.git
-	branch = trunk
+	url = ../cassandra-accord.git
+	branch = CASSANDRA-20608
```

Comment:
I'm assuming this reverts back to `trunk` when the Accord side merges...

Reply:
its done this way to make sure i can run CI